CrazyCanvas/Source/CrazyCanvas.cpp

```diff
@@ -200,3 +209,14 @@ bool CrazyCanvas::LoadRendererResources()
 
 	return true;
 }
+
+bool CrazyCanvas::BindComponentTypeMasks()
+{
+	using namespace LambdaEngine;
+
+	EntityMaskManager::BindTypeToExtensionDesc(WeaponLocalComponent::Type(), { 0 }, false);	// Bit = 0xF
+
+	EntityMaskManager::Finalize();
+
+	return true;
+}
```

Comment:
I'm not quite sure what this does, and if you don't mind explaining it to me, I'd like to know ^^